dolphinscheduler-alert/src/test/java/org/apache/dolphinscheduler/alert/plugin/AlertPluginManagerTest.java

```diff
@@ -53,6 +54,14 @@ public void testLoadPlugins() {
             alertPluginManagerConfig.setMavenLocalRepository(Objects.requireNonNull(PropertyUtils.getString(AlertServer.MAVEN_LOCAL_REPOSITORY)).trim());
         }
 
+
+        DolphinPluginLoader alertPluginLoader = new DolphinPluginLoader(alertPluginManagerConfig, ImmutableList.of(alertPluginManager));
+        try {
+            alertPluginLoader.loadPlugins();
+        } catch (Exception e) {
+            throw new RuntimeException("load Alert Plugin Failed !", e);
+        }
+
```

Comment:
It is best to do this, `public void testMethod() throws MyException {` 
No need to use try-catch to catch exceptions.

Reply:
Thanks for your advice ! @CalvinKirs https://github.com/apache/dolphinscheduler/pull/6674/commits/341d9aa50652da02ef85766390388f9a2c3c1ac0